measures/2018/measures-data.json

```diff
@@ -44942,7 +44942,7 @@
       "pediatrics"
     ],
     "measureSpecification": {
-      "registry": "https://qpp.cms.gov/docs/QPP_quality_measure_specifications/Claims-Registry-Measures/2018_Measure_467_Registry.pdf"
+      "default": "https://qpp.cms.gov/docs/QPP_quality_measure_specifications/Claims-Registry-Measures/2018_Measure_467_Registry.pdf"
```

Comment:
cc @sarahnw

Reply:
same issue with the misspaced columns. fixed on latest on task in jira.